src/http-servers/html-server.js

```diff
@@ -0,0 +1,12 @@
+import http from "http";
```

Comment:
Don't sure that it works. I don't get any response. No errors. Nothing
Also, are you going to implement the same but with stream?